### PR TITLE
Fix an indexing issue in build.cake

### PR DESCRIPTION
**Pull request by Phrohdoh**
This depends on #367 because you want that too. :-)

### CODE REVIEW

File: build.cake [hunk 1]
```diff
@@ -69,31 +69,39 @@ Task("deps").Does(() => {
     for (var i = 0; i < missingDepsCopy.Length; i++)
     {
         var dep = missingDepsCopy[i];
-        var depPathInOpenRA = depsInOpenRA[i];
+        var depPathInOpenRA = depsInOpenRA.SingleOrDefault(d => d.EndsWith(dep));
+        if (depPathInOpenRA == null)
+            throw new Exception(nameof(depsInOpenRA) + " does not contain an entry for " + dep);
```
Comment (Phrohdoh) [inline build.cake]: @abcdefg30 Please let me know if this fails to run for you.

I can remove the `nameof` easily I was just being lazy.

**Comment by GraionDilach** [comment]
The issue with cake is that it is included so I don't see this as a fix.

**Comment by Phrohdoh** [comment]
@GraionDilach You are more than welcome to submit a cross-platform solution to fetching dependencies, building, and cleaning.

**Comment by GraionDilach** [comment]
There was a solution which you prominently overwrote, along with my efforts to actually contribute into this repository.

**Comment by abcdefg30** [comment]
#386 was merged and `build.cake` removed. We could still use the first commit though.

**Comment by abcdefg30** [comment]
No activity.

**Pull request closed without merging**